composer.json

```diff
@@ -23,8 +23,9 @@
     "extra": {
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
-        "pimple/pimple": "^3.2.2"
+        "php": "^7.3 || ~8.0.0 || ~8.1.0",
+        "laminas/laminas-zendframework-bridge": "^1.0",
```

Comment:
Wait, what happened here? :| 

We should get rid of this, not add it...

Reply:
Upgraded `laminas-container-config-test` to version 0.4 and dropped `laminas/laminas-zendframework-bridge`.